gallery/src/styles/NxCard/NxCardPageExamples.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+// Reveal problems with box-sizing set to border-box, see https://issues.sonatype.org/browse/RSC-84
+
+.nx-card__content--2-column-example {
+  align-items: center;
+  display: flex;
+
+  >:first-child {
+    margin-right: 16px;
+  }
+}
+
+.nx-card--align-left {
+  text-align: left;
+}
+
+.nx-card-container--row .nx-card--75wide {
+  max-width: 75%;
+}
+
+.nx-card-container--row .nx-card--50wide {
+  max-width: 50%;
+}
+
+.nx-icon--indicator {
```

Comment:
This seems out of scope.  Also they aren't properly aligned

Reply:
> This seems out of scope. Also they aren't properly aligned

It was just meant to show how to do what has become a common layout in the mockups, when we have the status indicators in the RSC these should be replaced. I fixed the alignment issues here: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/17a7663a804fa121012ada6de25f21f4a2a1837c

Reply:
> It was just meant to show how to do what has become a common layout in the mockups, when we have the status indicator component in the RSC these should be replaced. I fixed the alignment issues here: 17a7663

I'd rather they just be removed until they are proper components.  They're just inline content so they don't really demonstrate anything special here, and they still aren't properly aligned.  In my experimentation on the Firewall page PR, I had to use a magic number to align them properly.

Reply:
> > It was just meant to show how to do what has become a common layout in the mockups, when we have the status indicator component in the RSC these should be replaced. I fixed the alignment issues here: [17a7663](https://github.com/sonatype/sonatype-react-shared-components/commit/17a7663a804fa121012ada6de25f21f4a2a1837c)
> 
> I'd rather they just be removed until they are proper components. They're just inline content so they don't really demonstrate anything special here, and they still aren't properly aligned. In my experimentation on the Firewall page PR, I had to use a magic number to align them properly.

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/fc2308d74662eb8447c2eabfa5446fca1b879174